NewRelicBrowser.php

```diff
@@ -1,392 +1,456 @@
-							'relic_app_script' => $browser_script,
-						);
-						add_option( $app_option_name, $browser_array );
-						add_settings_error( 'relic_options', 'relic_options_error', __( 'New Relic Browser App integrated successfully', 'rt-new-relic' ), 'updated' );
+							$browser_array = array(
+								'relic_app_name'   => $browser_list[0]->name,
+								'relic_app_key'    => $browser_list[0]->browser_monitoring_key,
+								'relic_app_id'     => $browser_list[0]->id,
+								'relic_app_script' => $browser_script,
+							);
+							add_option( $app_option_name, $browser_array );
+							add_settings_error( 'relic_options', 'relic_options_error', esc_html__( 'New Relic Browser App integrated successfully', 'rt-new-relic' ), 'updated' );
 
-						/* browser details saved so delete the browsers list from meta */
+							/* browser details saved so delete the browsers list from meta */
 
-						delete_option( $browser_app_list_option );
-					}
-				} else if ( 'rtp-create-browser' == $_POST['rtp-relic-form-name'] ) {
+							delete_option( $browser_app_list_option );
+						}
+					} elseif ( 'rtp-create-browser' === $_POST['rtp-relic-form-name'] ) {
 
-					/* We have api key. Create browser application. */
+						/* We have api key. Create browser application. */
 
-					$rtp_account_details = get_option( $option_name );
-					$account_api_key = $rtp_account_details['relic_api_key'];
-					if ( isset( $_POST['rtp-relic-browser-name'] ) ) {
-						$browser_created = $this->rtp_create_browser_app( sanitize_text_field( $_POST['rtp-relic-browser-name'] ), $account_api_key );
-					}
-					/* store the account details */
-					if ( $browser_created ) {
-						$account_details_array = array(
-							'relic_api_key' => $account_api_key,
-						);
-						add_option( $option_name, $account_details_array );
-						add_settings_error( 'relic_options', 'relic_options_error', __( 'New Relic Browser App integrated successfully', 'rt-new-relic' ), 'updated' );
-					}
-				} else if ( 'rtp-get-browser' == $_POST['rtp-relic-form-name'] ) {
-					/* check to see if option already exists */
-					$is_get_browser_exists = get_option( 'rtp_relic_browser_list',false );
-					if ( ! $is_get_browser_exists ) {
-						/* get the list of browser apps */
-						if ( isset( $_POST['rtp-user-api-key'] ) ) {
-							$account_api_key = sanitize_text_field( $_POST['rtp-user-api-key'] );
+						$rtp_account_details = get_option( $option_name );
+
+						$account_api_key = $rtp_account_details['relic_api_key'];
+						if ( isset( $_POST['rtp-relic-browser-name'] ) ) {
+							$browser_created = $this->rtp_create_browser_app( sanitize_text_field( wp_unslash( $_POST['rtp-relic-browser-name'] ) ), $account_api_key );
 						}
+						/* store the account details */
+						if ( $browser_created ) {
+							$account_details_array = array(
+								'relic_api_key' => $account_api_key,
+							);
+							add_option( $option_name, $account_details_array );
```

Comment:
Use `update_option()`

Reply:
i solved this issue. please review changes.